src/api/channels.tsx

```diff
@@ -126,6 +172,14 @@
         ) : (
           <React.Fragment>{String(value ?? '')}</React.Fragment>
         );
+    } else {
```

Comment:
Just as a note here, this should be fine assuming there are only 3 types of channel metadata and only scalar type is non-image.